### PR TITLE
use_tidy_github_actions()

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

**Comment by hadley** [comment]
Should fix #1552

File: .github/workflows/pkgdown.yaml [hunk 1]
```diff
@@ -1,21 +1,19 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/master/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    branches:
-      - main
-      - master
+    branches: [main, master]
+    tags: ['*']
```
Comment (maelle) [inline .github/workflows/pkgdown.yaml:6]: Should this be removed?

**Review by maelle — COMMENTED** [review]
LGTM (although I will be more confident after the next PR after this has been merged). Just a small question.

File: .github/workflows/pkgdown.yaml (hunk 1 above)
Comment (hadley) [reply .github/workflows/pkgdown.yaml:6]: Why? I think we want the site to be re-built when we push a tag (which indicates a release)

Comment (maelle) [reply .github/workflows/pkgdown.yaml:6]: ok!

**Pull request merged**